proxy-libs/helioslogrus/helioslogrus_test.go

```diff
@@ -0,0 +1,44 @@
+package helioslogrus
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"testing"
+
+	"encoding/json"
+
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/propagation"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/sdk/trace/tracetest"
+)
+
+func TestLogInstrumentation(t *testing.T) {
+	sr := tracetest.NewSpanRecorder()
+	tp := sdktrace.NewTracerProvider(sdktrace.WithSpanProcessor(sr))
+	otel.SetTracerProvider(tp)
+	propagator := propagation.NewCompositeTextMapPropagator(propagation.TraceContext{}, propagation.Baggage{})
+	otel.SetTextMapPropagator(propagator)
+	tracer := tp.Tracer("TestLogrus")
+	ctx, span := tracer.Start(context.Background(), "hello-span")
+
+	l := logrus.New()
```

Comment:
Why aren't we patching logsrus.New?

Reply:
We can but it won't be enough cause it's possible to write log without New
We can add it  as another method to helioslogrus (and not do the whole proxy-lib thing) wdyt?

Reply:
True, but why not handle the case where `New` is called? The other case can be "handled" but adding docs

Reply:
True, but why not handle the case where New is called? The other case can be "handled" but adding docs